src/components/KAlert/KAlert.vue

```diff
@@ -368,227 +373,234 @@ export const appearances: AlertAppearanceRecord = {
 
       &:not(:first-of-type) {
-        margin-left: var(--spacing-sm);
+        margin-left: var(--spacing-sm, var(--kui-space-50, $kui-space-50));
       }
     }
 
     &.info button.primary {
-      --KButtonPrimaryBase: var(--blue-500);
-      --KButtonPrimaryHover: var(--blue-200);
-      background-color: var(--blue-100);
-      color: var(--blue-500);
+      --KButtonPrimaryBase: var(--blue-500, var(--kui-color-background-primary, $kui-color-background-primary));
+      --KButtonPrimaryHover: var(--blue-200, var(--kui-color-background-primary-weaker, $kui-color-background-primary-weaker));
+      background-color: var(--blue-100, var(--kui-color-background-primary-weakest, $kui-color-background-primary-weakest));
+      color: var(--blue-500, var(--kui-color-text-primary, $kui-color-text-primary));
     }
 
     &.info button.outline {
-      --KButtonOutlineBorder: var(--blue-500);
-      --KButtonOutlineHoverBorder: var(--blue-600);
-      --KButtonOutlineActive: var(--blue-100);
-      --KButtonOutlineActiveBorder: var(--blue-500);
-      border: 1px solid var(--blue-400);
-      color: var(--blue-500);
+      --KButtonOutlineBorder: var(--blue-500, var(--kui-color-border-primary, $kui-color-border-primary));
+      --KButtonOutlineHoverBorder: var(--blue-600, var(--kui-color-border-primary-strong, $kui-color-border-primary-strong));
+      --KButtonOutlineActive: var(--blue-100, var(--kui-color-background-primary-weakest, $kui-color-background-primary-weakest));
+      --KButtonOutlineActiveBorder: var(--blue-500, var(--kui-color-border-primary, $kui-color-border-primary));
+      border: 1px solid var(--blue-400, var(--kui-color-border-primary-weak, $kui-color-border-primary-weak));
+      color: var(--blue-500, var(--kui-color-text-primary, $kui-color-text-primary));
     }
 
     &.warning button.primary {
-      --KButtonPrimaryBase: var(--yellow-500);
-      --KButtonPrimaryHover: var(--yellow-200);
-      background-color: var(--yellow-100);
-      color: var(--yellow-500);
+      --KButtonPrimaryBase: var(--yellow-500, $tmp-color-yellow-500);
+      --KButtonPrimaryHover: var(--yellow-200, $$tmp-color-yellow-200);
+      background-color: var(--yellow-100, $tmp-color-yellow-100);
+      color: var(--yellow-500, $tmp-color-yellow-500);
     }
 
     &.warning button.outline {
-      --KButtonOutlineBorder: var(--yellow-500);
-      --KButtonOutlineHoverBorder: var(--yellow-500);
-      --KButtonOutlineActive: var(--yellow-100);
-      --KButtonOutlineActiveBorder: var(--yellow-500);
-      border: 1px solid var(--yellow-300);
-      color: var(--yellow-500);
+      --KButtonOutlineBorder: var(--yellow-500, $tmp-color-yellow-500);
+      --KButtonOutlineHoverBorder: var(--yellow-500, $tmp-color-yellow-500);
+      --KButtonOutlineActive: var(--yellow-100, $tmp-color-yellow-100);
+      --KButtonOutlineActiveBorder: var(--yellow-500, $tmp-color-yellow-500);
+      border: 1px solid var(--yellow-300, $tmp-color-yellow-300);
```

Comment:
question: can't `1px` here be replaced with a token?

Same comment for all similar instances

Reply:
Good catch, fixed